tn_api_spec.yml

```diff
@@ -770,7 +766,7 @@ components:
             description: "Arrival date and time with timezone."
             example: "2020-12-06T09:10:00+00:00"
           flight_number:
-            type: integer
+            type: string
             description: Flight number.
             example: 530
```

Comment:
I added this as i noticed the type we return is string, however it might make more sense to be a int type. In which case i will revert and maybe make a ticket for next sprint where we could address any discrepancies that are found.